src/GameInternal.cpp

```diff
@@ -62,11 +62,11 @@ void GameInternal::init(const char* title, int xpos, int ypos, int width, int he
 		return;
 	}
 
-    SDL_Surface* icon = SDL_LoadBMP("assets/iconImage.bmp");
-    if(!icon)
+    // bad
+    SDL_Surface* icon;
+    if((icon = SDL_LoadBMP("assets/iconImage.bmp")))
```

Comment:
Requires: #70
Until then the game dev is required to use this name for the icon